vars/common-vars.yml

```diff
@@ -64,6 +66,44 @@ suse_airship_components_image_tag: "{%- if suse_airship_build_local_images or su
 
 socok8s_site_name: "soc"
 
+# suffix used in FQDN for ingress (i.e. "publicly facing" access point)
```

Comment:
should document the new options in the deployment guide.

Reply:
can do once docs branch is merged.